src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -69,434 +54,55 @@ namespace Microsoft.Identity.Client
-                .ExecuteAsync(CancellationToken.None).ConfigureAwait(false);
-        }
-
-        /// <summary>
-        /// Interactive request to acquire token for a given account, with the possibility of controlling the user experience, passing extra query
-        /// parameters, providing extra scopes that the user can pre-consent to, and overriding the authority pre-configured in the application
-        /// </summary>
-        /// <param name="scopes">Scopes requested to access a protected API</param>
-        /// <param name="account">Account to use for the interactive token acquisition. See <see cref="IAccount"/> for ways to get an account</param>
-        /// <param name="prompt">Designed interactive experience for the user.</param>
-        /// <param name="extraQueryParameters">This parameter will be appended as is to the query string in the HTTP authentication request to the authority.
-        /// This is expected to be a string of segments of the form <c>key=value</c> separated by an ampersand character.
-        /// The parameter can be null.</param>
-        /// <param name="extraScopesToConsent">scopes that you can request the end user to consent upfront, in addition to the scopes for the protected Web API
-        /// for which you want to acquire a security token.</param>
-        /// <param name="authority">Specific authority for which the token is requested. Passing a different value than configured does not change the configured value</param>
-        /// <param name="parent">Object containing a reference to the parent window/activity. REQUIRED for Xamarin.Android only.</param>
-        /// <returns>Authentication result containing a token for the requested scopes and account</returns>
-        public async Task<AuthenticationResult> AcquireTokenAsync(IEnumerable<string> scopes, IAccount account,
-                                                                  Prompt prompt, string extraQueryParameters, IEnumerable<string> extraScopesToConsent, string authority, UIParent parent)
-        {
-            GuardNetCore();
-
-            return await AcquireTokenInteractive(scopes, GetParentObjectFromUiParent(parent))
-                .WithAccount(account)
-                .WithPrompt(prompt)
-                .WithExtraQueryParameters(extraQueryParameters)
-                .WithExtraScopesToConsent(extraScopesToConsent)
-                .WithAuthority(authority)
-                .WithUseEmbeddedWebView(parent.CoreUIParent.UseEmbeddedWebview)
-                .ExecuteAsync(CancellationToken.None).ConfigureAwait(false);
-        }
-
-        private void GuardNetCore()
+            object parent)
         {
 #if NET_CORE
             throw new PlatformNotSupportedException("On .NET Core, interactive authentication is not supported. " + 
                 "Consider using Device Code Flow https://aka.ms/msal-net-device-code-flow or Integrated Windows Auth https://aka.ms/msal-net-iwa");
```

Comment:
>Consider using Device Code Flow [](start = 17, length = 31)

Should we suggest alto that they look at bringing their own browser or using the system browser approach (though we don't have it implemented yet, but we will soon)?